src/components/PurchaseOrder/PO.js

```diff
@@ -242,10 +241,21 @@ class PO extends Component {
     const orderNumber = get(order, 'po_number', '');
     const poLines = get(order, 'po_lines', []);
     const workflowStatus = get(order, 'workflow_status');
-    const isCloseOrderButtonVisible = orderId && workflowStatus === WORKFLOW_STATUS.open;
+    const isWorkflowStatusOpen = workflowStatus === WORKFLOW_STATUS.open;
+    const isCloseOrderButtonVisible = orderId && isWorkflowStatusOpen;
+    const isReceiveButtonVisible = orderId && isWorkflowStatusOpen && (poLines.length > 0);
     const lastMenu = (
       <PaneMenu>
         <IfPermission perm="purchase_order.item.put">
+          {isReceiveButtonVisible && (
+            <Button
+              buttonStyle="primary"
+              marginBottom0
+              style={{ marginRight: '10px' }}
```

Comment:
these `marginRight` don't work well with RTL feature. Maybe it makes sense to use flebox spacing here between buttons